platform/cavium/docker-syncd-cavm/Dockerfile.template

```diff
@@ -0,0 +1,18 @@
+FROM docker-base
+
+RUN apt-get update
+
+COPY deps /deps
+
+RUN apt-get -y install libpcap-dev libxml2-dev python-dev swig libsensors4-dev
+
+SED_DPKG
```

Comment:
SED_DPKG is superceded by jinj2 template in commit [66aebb3](https://github.com/Azure/sonic-buildimage/commit/66aebb329ccec74e92ad25875ea3e8cc2de65055)